code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -714,12 +714,6 @@
 	icon_state = "creamsuit"
 	item_state = "creamsuit"
 
-/obj/item/clothing/under/stripper/mankini
```

Comment:
You remove the mankini here, does it exist elsewhere in the code? otherwise i’m not sure where it’s item state would be defined! :)

Reply:
`/obj/item/clothing/under/stripper/mankini` existed in such a appropriately named jumpsuit location, which was `code/modules/clothing/suits/miscellaneous.dm`... a file for suits, I have no clue why it was there, that's why I missed it with the DMI split

`/obj/item/clothing/under/costume/mankini` replaced it for all intents and purposes so `/obj/item/clothing/under/stripper/mankini` is no longer needed.